src/heretic/mn_menu.c

```diff
@@ -2542,10 +2600,12 @@ static void DrawCrispnessItem(boolean item, int x, int y)
     MN_DrTextA(item ? "ON" : "OFF", x, y);
 }
 
-static void DrawCrispnessMultiItem(int item, int x, int y, const multiitem_t *multi)
+static void DrawCrispnessMultiItem(int item, int x, int y, const multiitem_t *multi,
+        boolean cond)
 {
-    dp_translation = item ? cr[CR_GREEN] : cr[CR_DARK];
-    MN_DrTextA(multi[item].name, x, y);
+    dp_translation = cond ? cr[CR_GRAY] :
```

Comment:
So, if `cond == true` then the item is grayed out?

Reply:
The intent is to have this text be the same color as the preceding menu item name. It  appears that `CR_GRAY` maps to the default color (white) but I should probably use `NULL` here instead. Good catch.